config/config_wcoss_dell_p3.sh

```diff
@@ -25,3 +25,9 @@ export LMOD_EXTENDED_DEFAULT="yes"
 
 # Load these basic modules
 module load cmake/3.16.2
+
+# Load for MET
+module load python/3.6.3
```

Comment:
This means every library will have this module loaded and if it finds something in this path, it is going to be a dependency.
modules should be loaded where they are used unless something like cmake.

Note to @kgerheiser 
We should treat python modules as compilers and create a meta-module for loading python on different machines as they are named differently.
Lets discuss this offline.

Reply:
@aerorahul In an email, @kgerheiser requested that I put "module load python/3.6.3" in config_wcoss_dell_p3.sh instead of in build_met.sh. Along with the export variables mentioned in your next comment:
```
export MET_PYTHON=/usrx/local/prod/packages/python/3.6.3
export MET_PYTHON_CC=-I${MET_PYTHON}/include/python3.6m
export MET_PYTHON_LD=-L${MET_PYTHON}/lib/\ -lpython3.6m\ -lpthread\ -ldl\ -lutil\ -lm\ -Xlinker\ -export-dynami
```
This makes sense to me as well, because on WCOSS, one cannot simply run "module load python", a version has to be given:
```
[Julie.Prestopnik@m71a3 hpc-stack]$ module load python
Lmod has detected the following error:  These module(s) exist but cannot be loaded as requested: "python"
   Try: "module spider python" to see how to load the module(s).
```
There is no default, which would be reference with a "D" below:
```
[Julie.Prestopnik@m71a3 hpc-stack]$ module avail python

------------------------------------------------------- Third Party Applications and Compilers --------------------------------------------------------
   python/2.7.13    python/2.7.14    python/3.6.3 (L)
```
Since this config file is specific to WCOSS it seems like a good place for it to be, but of course, that is from my limited perspective.  You know this system much better than I do.  
It seems that moving the "module load python/3.6.3" and the export statements into build_met.sh would cause problems in loading python on other machines, as version 3.6.3 may not be available and the path for MET_PYTHON would not be the same on another machine.